packages/hop-node/src/theGraph/getBondedWithdrawals.ts

```diff
@@ -26,7 +31,20 @@ export default async function getBondedWithdrawals (chain: string, token: string
     }
   `
   const jsonRes = await makeRequest(chain, query, {
-    token
+    token,
+    lastId
   })
-  return jsonRes.withdrawalBondeds.map((x: any) => normalizeEntity(x))
+  let withdrawals = jsonRes.withdrawalBondeds.map((x: any) => normalizeEntity(x))
+
+  lastId = withdrawals[withdrawals.length - 1].id
```

Comment:
is it possible that `withdrawals.length` here can be `0` so `.id` would throw an error trying to read from `undefined`?

Reply:
Yes, great call. Fixed with 26ffbf5f93e5a0d1cd10ed6b2e87d8b1919e4177